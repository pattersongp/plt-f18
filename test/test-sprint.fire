func int main = () => {
    sprint("hello world!");
    sprint("100");
    
    str q = "hasn't anyone told you that shouldn't play with FIRE?");
    sprint(q);
    return 0;

}
