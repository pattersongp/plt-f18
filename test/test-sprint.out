hello world!
100
hasn't anyone told you that you shouldn't play with FIRE?
